app/components/post_body/post_body.js

```diff
@@ -193,6 +195,11 @@ class PostBody extends PureComponent {
             if (canDelete && !hasBeenDeleted) {
                 actions.push({text: formatMessage({id: 'post_info.del', defaultMessage: 'Delete'}), onPress: onPostDelete});
             }
+
+            actions.push({
+                text: formatMessage({id: 'post_info.mobile.add_reaction', defaultMessage: 'Add Reaction'}),
```

Comment:
Are you sure you're looking at the latest commit? The latest is `fcac53067c1b81c0a9f6c72b85be793c411d8d1f` and it's `mobile.post_info.add_reaction` now.